src/Databases/SQLite/fetchSQLiteTableStructure.cpp

```diff
@@ -31,13 +31,18 @@ static DataTypePtr convertSQLiteDataType(String type)
     DataTypePtr res;
     type = Poco::toLower(type);
 
+    /// The SQLite columns get the INTEGER affinity if the type name contains "int". This means variable-length integers up to 8 bytes. The bit width is not really enforced even
+    /// in a STRICT table, so in general we should treat these columns as Int64. Besides that, we allow some common fixed-width int specifiers for applications to select a 
+    /// particular width, even though it's not enforced in any way by SQLite itself.
+    /// Docs: https://www.sqlite.org/datatype3.html
+    /// The most insane quote from there: Note that a declared type of "FLOATING POINT" would give INTEGER affinity, not REAL affinity, due to the "INT" at the end of "POINT". 
```

Comment:
:+1:

Reply:
Let me join your decent comments with the https://github.com/ClickHouse/ClickHouse/pull/73853

Reply:
https://github.com/ClickHouse/ClickHouse/pull/73853/commits/212e75e4881b86cf4ffa9bf6e7d3fd17cecd2a3b